src/shogun-core-main/src/main/java/de/terrestris/shoguncore/util/interceptor/secure/WfsResponseInterceptor.java

```diff
@@ -180,6 +185,31 @@ public Response interceptDescribeFeatureType(MutableHttpServletRequest request,
         LOG.debug("Intercepting WFS DescribeFeatureType response");
         ByteArrayOutputStream bout = new ByteArrayOutputStream();
         try {
+            if (response.getHeaders().get("content-type").get(0).toLowerCase(Locale.ROOT).contains("json")) {
```

Comment:
Even if the `content-type` should be present always, I would suggest to include a check to prevent a potential NPE.

Reply:
NPE is catched, look at your smiley to find it